src/Symfony/Component/Security/Core/Authorization/Voter/Voter.php

```diff
@@ -24,10 +24,14 @@
  */
 abstract class Voter implements VoterInterface, CacheableVoterInterface
 {
-    public function vote(TokenInterface $token, mixed $subject, array $attributes): int
+    /**
+     * @param Vote $vote Should be used to explain the vote
```

Comment:
should or could ? (not native speaker tho)

Reply:
"should" to me